test/parse_source_test.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Test: parseSource
+ */
+var deepEqual = require('deep-equal');
+var apidoc = require("../lib/index");
+
+function log() {
+//   console.log(arguments);
```

Comment:
is this left commented on purpose? Do we even need to specify a logger in the tests?

Reply:
no,i just forget to delete this line, i will update later.
yes,because if we do not setLogger [this line](https://github.com/apidoc/apidoc-core/blob/8ead7e7ed9faa13c4f870a82ab22e0d17bc76561/lib/index.js#L176) will report error,because of #105